statement.k

```diff
@@ -16,17 +17,79 @@ rule if (false) S:Statement => .
 rule if (true) S:Statement else S1:Statement => #exeStmt(S)
 rule if (false) S:Statement else S1:Statement => #exeStmt(S1)
 
-rule #exeStmt({ Ss:Statements }) =>  Ss 
+rule #exeStmt({ Ss:Statements }) =>  Ss
+
+rule #exeStmt(S:IfStatement) => S
+
+rule #exeStmt(S:WhileStatement) => S
```

Comment:
If these only can apply on the `<k>` cell, then lets wrap them in `<k> ... </k>` tags.